source/lib/cdk-textract-stack.ts

```diff
@@ -932,6 +933,7 @@ export class CdkTextractStack extends cdk.Stack {
           DOCUMENTS_TABLE: documentsTable.tableName,
           ES_DOMAIN: elasticSearch.attrDomainEndpoint,
           PDF_LAMBDA: pdfGenerator.functionName,
+          ENABLE_COMPREHEND_MEDICAL: props.enableComprehendMedical.toString(),
```

Comment:
Is there need for the toString function here? I believe we can pass booleans to Lambda too and it would be considered as a string

Reply:
Yep. The toString function is needed. Lambda environment vars do not accept boolean values unless they are in the form of a string. https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html

Reply:
The type definitions also support this - type of property `environment` in `interface FunctionOptions` is 
```
readonly environment?: {
  [key: string]: string
}
```